src/components/map/ClusterMarker.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { ImageBackground, Text, View } from 'react-native';
+import PropTypes from 'prop-types';
+import { Marker } from 'react-native-maps';
+import returnClusterStyle from '../../modules/map/ClusterHelper';
+
+const CLUSTER_IMAGE = require('../../../assets/cluster.png');
+
+const ClusteredMarker = ({ id, geometry, properties, onPress }) => {
+  const points = properties.point_count;
+  const { width, height, top, fontSize } = returnClusterStyle(points);
+
+  return (
+    <Marker
+      key={`cluster-marker-${id}`}
+      coordinate={{
+        longitude: geometry.coordinates[0],
+        latitude: geometry.coordinates[1],
+      }}
+      onPress={onPress}
+    >
+      <ImageBackground source={CLUSTER_IMAGE} style={{ width, height }}>
+        <View
+          style={{
+            position: 'absolute',
+            top,
+            left: 0,
+            right: 0,
+            bottom: 0,
+            justifyContent: 'center',
+            alignItems: 'center',
+          }}
```

Comment:
Please use style sheet.

Reply:
Of course, this is how it should be done. I will fix that.